src/main/java/uk/gov/hmcts/reform/slc/model/LetterPrintStatus.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.slc.model;
+
+import java.time.ZonedDateTime;
+
+public class LetterPrintStatus {
```

Comment:
Would tracking the print status of the letter made more sense if it was kept with the Letter object?